product_multi_company/tests/test_product_multi_company.py

```diff
@@ -11,7 +11,6 @@
 class TestProductMultiCompany(ProductMultiCompanyCommon, common.TransactionCase):
     def test_create_product(self):
         product = self.env["product.product"].create({"name": "Test"})
-        self.assertFalse(product.company_id)
```

Comment:
Why this line is being removed? This means a change of behavior.

Reply:
Previously it was giving errors, now it isn't anymore